js/main.js

```diff
@@ -21,35 +21,31 @@ function setupPlayer() {
 	bitmovin.player.core.Player.addModule(window.bitmovin.player.style.default);
 	bitmovin.player.core.Player.addModule(window.bitmovin.player.tizen.default);
 
+	
+	var bufferConfig = {};
+	var bufferLevels = {};
+	bufferLevels[bitmovin.player.core.BufferType.ForwardDuration] = 30;
+	bufferLevels[bitmovin.player.core.BufferType.BackwardDuration] = 10,
+	bufferConfig[bitmovin.player.core.MediaType.Video] = bufferLevels;
+	bufferConfig[bitmovin.player.core.MediaType.Audio] = bufferLevels;
+	
 	var conf = {
 		key : "YOUR_PLAYER_KEY",
 		playback : {
 			autoplay : true,
-			preferredTech: [
```

Comment:
This section was just added by me in a PR. We have seen some devices try to use the native player if this is not specified, so it should be added back.

Reply:
We recently had a player bug, where Tizen (Webkit based) was detected as Safari, thus the Native route was used. This issue has been resolved, deeming this code unnecessary.

It's not wrong, it's just not needed anymore, so I thought it was a good idea to remove redundant code.

Reply:
Thanks for clarifying @bitmovin-kenny.